firebase-firestore/ktx/src/main/kotlin/com/google/firebase/firestore/ktx/annotations/KIgnoreExtraProperties.kt

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.firestore.ktx.annotations
+
+import kotlinx.serialization.SerialInfo
+
+/**
+ * Properties that don't map to class fields are ignored when serializing to a class annotated with
```

Comment:
You can add that this is our equivalent to JSON"s  `Json { ignoreUnknownKeys = true }`

Reply:
Also, from our conversation, you can specify that this is the default behaviour.

Reply:
I have added:

 1) this is the default behavior and 
2) this is the equivalent to `Json { ignoreUnknownKeys = true }` and current Java `@ThrowOnExtraproperties` annotation.